DANS-KNAW/easy-split-multi-deposit
# fixed and added to the unit tests that are impacted due to this change

fixes EASY-1779-1781-adds-identifier.doi.registered

#### When applied it will
* deposit.properties files will be created with property identifier.doi.registered=no

#### Where should the reviewer @DANS-KNAW/easy start?

The AddPropertiesToDeposit class

#### How should this be manually tested?

create a new deposit and check if the property is added to the deposit.properties file

#### related pull requests on github
